lib/cucumber/formatter/junit.rb

```diff
@@ -25,15 +25,16 @@ def initialize(config)
         config.on_event :test_run_finished, &method(:on_test_run_finished)
         @reportdir = ensure_dir(config.out_stream, 'junit')
         @config = config
-        @features_data = Hash.new { |h,k| h[k] = {
+        @features_data = Hash.new do |h,k| h[k] = {
```

Comment:
whilst not in this cop, probably worth doing `|h, k|` just whilst you're here. Also line 35 can go to new syntax if you're inclined

Reply:
All set on this too. :)